httpx/client.py

```diff
@@ -241,6 +243,12 @@ def merge_queryparams(
                 finally:
                     await response.close()
 
+            request_line = (
```

Comment:
Looks like is short enough to be on a single line according to black:

```suggestion
            request_line = f"{request.method} {request.url} {response.http_version}"
```